rubocop.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rainbow', '>= 2.2.2', '< 4.0')
   s.add_runtime_dependency('regexp_parser', '>= 1.8', '< 3.0')
   s.add_runtime_dependency('rexml', '>= 3.2.5', '< 4.0')
-  s.add_runtime_dependency('rubocop-ast', '>= 1.28.1', '< 2.0')
+  s.add_runtime_dependency('rubocop-ast', '>= 1.28.1', '< 2.0') # TODO: Needs BlockNode#{first,last}_argument
```

Comment:
This will need to be updated to whatever version of `rubocop-ast` releases rubocop/rubocop-ast#270 (likely 1.30.0).